hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ResponseTemplate.java

```diff
@@ -218,10 +218,10 @@ void ReadResultSet(int readCount, List<Data> items,
     @Response(ResponseMessageConst.ENTRIES_WITH_CURSOR)
     void EntriesWithCursor(int tableIndex, List<Map.Entry<Data, Data>> entries);
 
-    @Since("1.4")
+    @Since("1.6")
     @Response(ResponseMessageConst.SCHEDULED_TASK_STATISTICS)
     void ScheduledTaskStatistics(long lastIdleTimeNanos, long totalIdleTimeNanos,
-                                 long totalRuns, long totalRunTimeNanos);
+                                 long totalRuns, long totalRunTimeNanos, long lastRunDurationNanos);
```

Comment:
You need to add the tag to field like this: 
```
 @Since("1.4")
    @Response(ResponseMessageConst.SCHEDULED_TASK_STATISTICS)
    void ScheduledTaskStatistics(long lastIdleTimeNanos, long totalIdleTimeNanos,
                                 long totalRuns, long totalRunTimeNanos,  @Since("1.6") long lastRunDurationNanos);
```